benefits/static/css/styles.css

```
@@ -142,6 +142,25 @@ a:visited:not(.btn) {
   color: var(--selected-color);
 }
 
+[target="_blank"]::after {
```

I think at a minimum we want to add the element here:

```suggestion
a[target="_blank"]::after {
```

Is targeting all links with `_blank` enough? Too much? Is it consistent practice to make external links open in new tabs?

Asking Sarah some related questions on the Figma board:

* https://www.figma.com/file/SeSd3LaLd6WkbEYhmtKpO3?node-id=9234:110446&mode=design#500359107
* https://www.figma.com/file/SeSd3LaLd6WkbEYhmtKpO3?node-id=9234:110446&mode=design#500359437